dashboard.py

```diff
@@ -1,13 +1,15 @@
 """Runs the streamlit dashboard"""
 
 import logging
+from datetime import datetime
 import pandas as pd
 import altair as alt
 import streamlit as st
 import requests
 import psycopg2
 from bs4 import BeautifulSoup
-from streamlit_graphs import get_connection, get_cursor
+from database_connection import get_connection, get_cursor
+from streamlit_option_menu import option_menu
 
 headerSection = st.container()
```

Comment:
Very small detail, but you may want to change this to snake case to match over variable names.